app/Http/Controllers/CourseController.php

```diff
@@ -14,7 +14,7 @@ class CourseController extends Controller
     /**
      * @var CanvasService
      */
-    private $canvasService;
+    protected $canvasService;
```

Comment:
Access Modifier: The access modifier of the $canvasService property has been changed from private to protected. This may or may not be necessary depending on the use case of the class. private properties are not accessible outside the class, while protected properties are accessible to subclasses and classes in the same namespace. If this property is not being used outside the class, it may be better to keep it as private.